solutions/src/main/java/ru/sberbank/school/task13/cacheproxy/Cache.java

```diff
@@ -0,0 +1,21 @@
+package ru.sberbank.school.task13.cacheproxy;
+
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Cache {
+    enum cacheType {MEMORY, FILE}
```

Comment:
WhitespaceAround: &#39;}&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a579020240>*